Objects/typeobject.c

```diff
@@ -4070,10 +4070,24 @@ type_setattro(PyTypeObject *type, PyObject *name, PyObject *value)
 extern void
 _PyDictKeys_DecRef(PyDictKeysObject *keys);
 
+
+void
+_PyStaticType_Dealloc(PyTypeObject *type)
+{
+    // Deallocating a static type is experimental: don't do it
+    // in release builds
+#ifdef Py_DEBUG
+    assert(type->tp_subclasses == NULL);
+    Py_CLEAR(type->tp_dict);
```

Comment:
Can we deallocate the tp_members created on heap in here?
Ref: https://github.com/python/cpython/blob/7c0914d35eaaab2f323260ba5fe8884732533888/Objects/structseq.c#L499

Reply:
Oooooh, I had a note about this very specific member for a very long time and I forgot about it. Right, it can be deallocated!